apps/beets-frontend-v3/README.md

```diff
@@ -63,7 +63,7 @@ pnpm dev:beets:webpack
 
 ## Testing
 
-See [TESTING.md](./test/TESTING.md).
+See [TESTING.md](../../packages/lib/test/TESTING.md).
```

Comment:
thanks for the PR!

the link should actually be `See [Testing instructions](../../README.md#testing).`